tests/upload_test.go

```diff
@@ -1184,11 +1179,12 @@ var _ = Describe("CDIConfig manipulation upload tests", Serial, func() {
 		pvc = f.CreateBoundPVCFromDefinition(utils.UploadPVCDefinition())
 
 		By("Verify Quota was exceeded in logs")
-		matchString := "pods \\\"cdi-upload-upload-test\\\" is forbidden: exceeded quota: test-quota, requested"
-		Eventually(func() string {
-			log, err := f.RunKubectlCommand("logs", f.ControllerPod.Name, "-n", f.CdiInstallNs)
-			Expect(err).NotTo(HaveOccurred())
-			return log
+		matchString := `pods \"cdi-upload-upload-test\" is forbidden: exceeded quota: test-quota, requested`
+		Eventually(func() ([]byte, error) {
+			return f.K8s().CoreV1().
```

Comment:
is `f.K8s()` a thing?

Reply:
Yep, `Frameworks` embeds `Clients` which has a `K8s` method:

https://github.com/kubevirt/containerized-data-importer/blob/4a64af4bdb4d1215c71008dc6b9aca3c7e2c94f9/tests/framework/framework.go#L103-L105

Reply:
So I see we mostly use `f.K8sClient`, maybe want to align to that?

Reply:
Sounds good.